assume/units/powerplant.py

```diff
@@ -76,7 +74,7 @@ def __init__(
         downtime_warm_start: int = 48,  # hours
         heat_extraction: bool = False,
         max_heat_extraction: float = 0,
-        location: tuple[float, float] = (0.0, 0.0),
+        location: Tuple[float, float] = (0.0, 0.0),
```

Comment:
this change is the same for all supported versions, but I don't mind

Reply:
What do you mean by versions?